webapi/middleware.go

```diff
@@ -163,6 +164,37 @@ func broadcastTicket() gin.HandlerFunc {
 
 		dcrdClient := c.MustGet("DcrdClient").(*rpc.DcrdRPC)
 
+		if request.ParentHex != "" {
+			parentTx, err := decodeTransaction(request.ParentHex)
```

Comment:
Maybe `getTransaction` for the parent first? If it exists, we can skip the validation and SendRawTransaction.